test/stat_tracker_test.rb

```diff
@@ -99,4 +173,14 @@ def test_it_can_get_avg_goals_by_season
     }
     assert_equal seasons_and_avg_goals, @@stat_tracker.average_goals_by_season
   end
+
+  # ==================       League Stats Tests      ==================
+
+  def test_it_can_find_the_best_offesive_team
+    assert_equal "FC Dallas", @@stat_tracker.best_offense
+  end
+
```

Comment:
@guardiac based on the spreadsheet data, looks like this one is a tie between two teams. In our project check-in, we could ask how ties should be handled because it occurs in other stat methods too.

Reply:
Yes!! I have that written down as a question I want to ask them.